contributing-docs/quick-start-ide/contributors_quick_start_pycharm.rst

```diff
@@ -39,12 +39,30 @@ Setup your project
              alt="Cloning github fork to Pycharm">
       </div>
 
-3. Configure the source root directories for providers as well as for task_sdk.
+3. Configure the source root directories well as for task_sdk. You also have to
+   set "source" and "tests" root directories for each provider you want to develop (!).
+   This is important in Airflow 3.0 we split providers to be separate
+   distributions - each with separate ``pyproject.toml`` file, so you need to separately
```

Comment:
ahhh that is a lot of manual work :(

Reply:
If you want to develop all providers - yes. But this is a rare beast I think.

Reply:
VAAST majority of people wants to work on one provider at a time.